app/models/notifications/possible_duplicate_admin_notification.rb

```diff
@@ -0,0 +1,17 @@
+module Notifications
+  class PossibleDuplicateAdminNotification < ::NotificationType
+    def mailer_message
+      "Possible duplicate (#{subject_record}) entered by #{created_by_user}"
+    end
+
+    def description
+      'Possible duplicate found'
+    end
+
+    private
+
+      def created_by_user
+        subject_record.audits.where(action: 'create').first&.user
```

Comment:
suggest we raise an exception if this is ever blank, and do you think there could be a race condition where this is triggered before the audit is created?

Reply:
done, see other comment for info about race condition